src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -294,11 +294,6 @@ read_file_itk(const std::string &filename)
           typedef itk::GDCMSeriesFileNames NamesGeneratorType;
           typename NamesGeneratorType::Pointer nameGenerator = NamesGeneratorType::New();
           nameGenerator->SetUseSeriesDetails( true );
-          // Make sure we read only data from a single frame and gate
-          nameGenerator->AddSeriesRestriction("0008|0022" ); // AcquisitionDate
-          nameGenerator->AddSeriesRestriction("0008|0032" ); // AcquisitionTime
-          nameGenerator->AddSeriesRestriction("0018|1060" ); // TriggerTime
-          nameGenerator->AddSeriesRestriction("0018|1063" ); // FrameTime
```

Comment:
@KrisThielemans 

I found that when reading DICOM composed-multi-bed-position images, the times are different depending on the slice. But I'd have thought that the default name generator arguments would be sufficient in most cases - although I haven't tested on any dynamic data (which I suspect is the reason for this).

Reply:
NB: reading composed images is probably not a common use-case so if this conflicts with frames of dynamic images then I can revert this change
NB NB: Perhaps this actually presents an issue if you try to read a dynamic DICOM data @rijobro

Reply:
At some point, we need to generalise this class to be able to read dynamic (or gated) data, with correct time frame info being returned. However, at the moment, this function is used for a single time frame (or gate). without the restriction, if the user has multiple gates/frames in the directory, the STIR image is going to be nonsensical (i.e. incorrect).

It's of course tricky what to do with "composed" images (i.e. data reconstructed from multiple-bed positions). The time frame info that we'd return is going to be incorrect in any case.

I feel this is a complication we don't really want to handle right now. I therefore would like to re-instate the restrictions. This way, the user can tell the reader to read a specific time-frame/gate (although in an awkward way).

It's a general problem with the "nice" reading infrastructure that sometimes you'd want to be able to pass options to the reader but we cannot, unless we'd stick options into the filename somehow. But that's for another PR...